runtime/onert/frontend/base_loader/include/base_loader.h

```diff
@@ -1804,89 +1250,113 @@ void BaseLoader<LoaderDomain, SpecificLoader>::loadOperation(const Operator *op,
       loadComparison(op, subg);
       return;
     case BuiltinOperator::BuiltinOperator_ONE_HOT:
-      loadOneHot(op, subg);
-      return;
-    case BuiltinOperator::BuiltinOperator_ABS:
-      loadElementwiseUnary(op, subg, ir::operation::ElementwiseUnary::Type::ABS);
-      return;
-    case BuiltinOperator::BuiltinOperator_COS:
-      loadElementwiseUnary(op, subg, ir::operation::ElementwiseUnary::Type::COS);
-      return;
-    case BuiltinOperator::BuiltinOperator_SIN:
-      loadElementwiseUnary(op, subg, ir::operation::ElementwiseUnary::Type::SIN);
+    {
+      if (op->inputs()->size() != 4 || op->outputs()->size() != 1)
+        throw std::runtime_error("OneHot Op has wrong number of input or output tensors.");
+
+      ir::operation::OneHot::Param param;
+      param.axis = op->builtin_options_as_OneHotOptions()->axis();
+
+      loadOperationTo<ir::operation::OneHot>(op, subg, param);
       return;
+    }
     case BuiltinOperator::BuiltinOperator_SHAPE:
-      loadShape(op, subg);
-      return;
-    case BuiltinOperator::BuiltinOperator_REDUCE_PROD:
-      loadReduce<ir::operation::Reduce::ReduceType::PROD>(op, subg);
+      // ir::operation::Shape::Param param;
```

Comment:
Is this commented out intentionally?

Reply:
This comment already existed in the original code. I don't know why this didn't commented out in detail. But It seems unnecessary. I will remove it.